recognition/include/pcl/recognition/3rdparty/metslib/simulated-annealing.hh

```diff
@@ -51,12 +51,12 @@ namespace mets {
   public:
     /// @brief Constructor
     abstract_cooling_schedule() 
-    { }
+    = default;
```

Comment:
2 more here

Reply:
I wonder why the hell some of these didn't show up in my find window?